cmd/antrea-agent/agent.go

```diff
@@ -401,6 +402,11 @@ func run(o *Options) error {
 			serviceCIDRNet)
 	}
 
+	var ipsecCertController *ipseccertificate.Controller
```

Comment:
I thought I added this comment earlier but seems not - I feel still better to have a feature gate (e.g. IPsecCertAuth) for both controller and agent code.

Reply:
Added feature gate `IPsecCertAuth`. Thanks!